extra/modules/pb-request-correction/src/main/java/org/prebid/server/hooks/modules/pb/request/correction/core/util/VersionUtil.java

```diff
@@ -0,0 +1,35 @@
+package org.prebid.server.hooks.modules.pb.request.correction.core.util;
+
+public class VersionUtil {
+
+    public static boolean isVersionLessThan(String versionAsString, int major, int minor, int patch) {
+        return compareVersion(versionAsString, major, minor, patch) < 0;
+    }
+
+    private static int compareVersion(String versionAsString, int major, int minor, int patch) {
```

Comment:
Maybe it's time to create some PBS util on core level?

Reply:
It`s too specific to be added to core utils.